Fix discrepency

DannyParker0001 opened the pull request:
Hi BowDown097,

Whilst using RRBot I noticed that two users, Danny Parker and Cashmere a Rolex were getting noticeably less successful gambles than other users. I believe you may have some uncommited changes that led to this.
Upon further investigation, there seemed to be no check to ensure these users have lower rates, so I added them in this PR.  

Kind regards,
Danny

DannyParker0001 committed:
Fix discrepency 

This might compile

binex-dsk commented on Modules/Gambling.cs:
```diff
@@ -181,6 +181,9 @@ private async Task<RuntimeResult> GenericGamble(double bet, double odds, double
         if (user.Perks.ContainsKey("Speed Demon"))
             odds *= 0.95;
+        // If danny or cashmere
+        if (Context.User.Id == 212496184776851456 || Context.User.Id == 294538745296846858)
```
This is missing user ID `874458978577428551`. Please fix this immediately and merging will be considered.

DannyParker0001 committed:
FIX: User in rngfix was not accounted for 

This patch adds swirl <@874458978577428551> to the rngfix to more accurately generate correct gambling results

Realswitzer reviewed (changes requested):
Create Venezuela (make it `odds *= 10000.5`)

DannyParker0001 commented:
> Create Venezuela (make it `odds *= 10000.5`)

Sorry this is currently held back by #7, perhaps this should be added in a new PR?

Realswitzer commented:
oh k, will review again

Realswitzer reviewed (approved):
ok it workie

DannyParker0001 commented:
@BowDown097 Can we get an ETA on the merge?

DannyParker0001 committed:
fix merge conflict

DannyParker0001 committed:
Merge branch 'BowDown097-master'
merge fixes merge conflicts with upstream
bowdown I love you

binex-dsk commented:
@BowDown097 This LGTM. Can we get a merge? This is needed ASAP as it fixes several long standing bugs and issues with the bot.
